cclib/bridge/cclib2horton.py

```diff
@@ -126,7 +124,26 @@ def makecclib(iodat):
                 attributes["atomcharges"]["natural"] = iodat.npa_charges
         elif hasattr(iodat, "npa_charges"):
             attributes["atomcharges"] = {"natural": iodat.npa_charges}
-    elif hortonver == 3:
-        pass
 
+    elif hortonver == 3:
+        # Horton 3 IOData class uses attr and does not have __dict__.
+        # In horton 3, some attributes have a default value of None.
+        # Therefore, second hasattr statement is needed for mo attribute.
+        if hasattr(iodat, "atcoords"):
+            # cclib parses the whole history of coordinates in the list, horton keeps the last one.
+            attributes["atomcoords"] = [iodat.atcoords]
+        if hasattr(iodat, "mo") and hasattr(iodat.mo, "norba"):
+            # MO coefficient should be transposed to match the dimensions.
+            attributes["mocoeffs"] = [iodat.mo.coeffs[: iodat.mo.norba].T]
+            if iodat.mo.kind == "unrestricted":
+                attributes["mocoeffs"].append(iodat.mo.coeffs[iodat.mo.norba :].T)
+        if hasattr(iodat, "spinpol"):
+            # IOData stores 2S, ccData stores 2S+1.
+            attributes["mult"] = iodat.spinpol + 1
+        if hasattr(iodat, "atnums"):
+            attributes["atnums"] = iodat.atnums
+        if hasattr(iodat, "atcorenums"):
+            attributes["coreelectrons"] = iodat.atnums - iodat.atcorenums
```

Comment:
I'm not familiar with Horton, but `atcorenums` sounds like it could be the number of core electrons. Is that correct?

Reply:
Yes, in `horton`, it looks like # of electrons after applying the pseudopotentials are stored (from what is parsed using its `from_file` method). `cclib` stores # of electrons that are approximated out with the pseudopotentials. I've added a comment right above the line to reduce confusion. Thanks!